### PR TITLE
feat(application): Implemented Installation API

### PR DESCRIPTION
**Pull request by OmAximani0**
- fixes: #140

### CODE REVIEW

File: crowdin_api/api_resources/application/resource.py [hunk 1]
```diff
@@ -19,6 +23,85 @@ class ApplicationResource(BaseResource):
     def get_application_path(self, applicationIdentifier: str, path: str):
         return f"applications/{applicationIdentifier}/api/{path}"
 
+    def get_application_installtions_path(self, identifier: Optional[str] = None):
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/application/resource.py:26]:
```suggestion
    def get_application_installations_path(self, identifier: Optional[str] = None):
```

File: crowdin_api/api_resources/application/enums.py
Comment (andrii-bodnar) [inline crowdin_api/api_resources/application/enums.py:1]: rename the `application` directory -> `applications`

File: crowdin_api/api_resources/application/types.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    value: ProjectPermission
+    ids: Iterable[int]
+
+
+class ApplicationPermissions(TypedDict):
+    user: ApplicationUser
+    project: ApplicationProject
+
+
+class ApplicationInstallionPatchRequest(TypedDict):
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/application/types.py:24]:
```suggestion
class ApplicationInstallationPatchRequest(TypedDict):
```

File: crowdin_api/api_resources/application/tests/test_applications_resources.py [hunk 3]
```diff
@@ -22,6 +24,140 @@ def test_get_applications_path(self, in_params, path, base_absolut_url):
...
+            params=request_param
+        )
+
+    @mock.patch("crowdin_api.requester.APIRequester.request")
+    def test_edit_application_installation(
+        self, m_request, base_absolut_url
+    ):
+        m_request.return_value = "response"
+
+        identifier = "exmaple-applcation"
```
Comment (andrii-bodnar) [inline crowdin_api/api_resources/application/tests/test_applications_resources.py:148]:
```suggestion
        identifier = "example-application"
```

**Review by andrii-bodnar — CHANGES_REQUESTED** [review]

File: crowdin_api/api_resources/application/enums.py
Comment (andrii-bodnar) [reply crowdin_api/api_resources/application/enums.py:1]: @OmAximani0 sorry, I didn't notice that it was already named this way. Please rollback this renaming

Comment (OmAximani0) [reply crowdin_api/api_resources/application/enums.py:1]: No worries, I'll rollback

**Review by andrii-bodnar — APPROVED** [review]
@OmAximani0 thank you!

**Pull request merged**